pontoon/administration/management/commands/commit_projects.py

```diff
@@ -17,7 +16,7 @@ class Command(BaseCommand):
 
     def handle(self, *args, **options):
         def output(text):
-            now = datetime.datetime.now()
+            now = datetime.now()
             self.stdout.write('[%s]: %s\n' % (now, text))
 
```

Comment:
We can probably get rid of the output() method, because Papertrail is adding timestamps anyways.